controllers/pxc_config.go

```diff
@@ -159,3 +164,41 @@ wsrep_trx_fragment_unit = bytes
 wsrep-provider-options = evs.delayed_keep_period=PT560S;evs.stats_report_period=PT1M;gcs.fc_limit=128;gmcast.peer_timeout=PT15S;gmcast.time_wait=PT18S;evs.max_install_timeouts=5;pc.recovery=true;gcache.recover=yes;gcache.size=8989366809;evs.delay_margin=PT30S;evs.user_send_window=1024;evs.inactive_check_period=PT5S;evs.join_retrans_period=PT5S;evs.suspect_timeout=PT60S;gcs.max_packet_size=131072;pc.linger=PT60S;evs.send_window=1024;evs.inactive_timeout=PT120S;pc.announce_timeout=PT60S;
 	`
 )
+
+var ( //nolint:dupl
+	// A pxcResourceRequirementsSmall is the resource requirements for PXC for small clusters.
+	pxcResourceRequirementsSmall = corev1.ResourceRequirements{
+		Requests: corev1.ResourceList{
+			corev1.ResourceMemory: resource.MustParse("1.61Gi"),
+			corev1.ResourceCPU:    resource.MustParse("570m"),
+		},
+		Limits: corev1.ResourceList{
+			corev1.ResourceMemory: resource.MustParse("1.6Gi"),
```

Comment:
the limit is smaller then the request, I believe it should be 1.7Gi (1825361100 bytes = 1.699999999255Gi)

Reply:
You're right, that's a typo. Thanks for catching it!